test/test_unistd.rs

```diff
@@ -78,8 +79,8 @@ fn test_mkstemp() {
 
 #[test]
 fn test_getpid() {
-    let pid = getpid();
-    let ppid = getppid();
+    let pid: ::libc::pid_t = getpid().into();
```

Comment:
I think the API is fine for normal use.  The only reason for the cast is for the assertion.  I don't think ordinary consumers of `getpid` will ever need the cast.